tests/integration/ceph_multi_cluster_test.go

```diff
@@ -108,12 +110,74 @@ func (mrc *MultiClusterDeploySuite) TearDownSuite() {
+	var args []string
+	switch deviceType {
+	case sys.DiskType:
+		args = []string{installer.TestScratchDevice(), sys.DiskType}
+	case sys.LVMType:
+		args = []string{installer.TestScratchDevice(), sys.LVMType}
+	default:
+		mrc.op.T().Fail()
+		mrc.op.installer.GatherAllRookLogs(mrc.op.T().Name(), mrc.op.namespace1, mrc.op.systemNamespace)
+		logger.Errorf("invalid device type %+v", deviceType)
+		return
+	}
+
+	cmdArgs := utils.CommandArgs{Command: localPathPVForOSDCmd, CmdArgs: args}
+	cmdOut := utils.ExecuteCommand(cmdArgs)
+	require.NoError(mrc.op.T(), cmdOut.Err)
+
+	_, err := mrc.k8sh.Kubectl("-n", mrc.namespace1, "patch",
+		"CephCluster", mrc.op.installer.ClusterName,
+		"--type=json", "-p", `[{"op": "replace", "path": "/spec/storage/storageClassDeviceSets/0/count", "value":1}]`)
+	require.NoError(mrc.op.T(), err)
+	mrc.createPools()
+}
+
+func (mrc *MultiClusterDeploySuite) deleteOSD() {
+	mrc.deletePools()
+	_, err := mrc.k8sh.Kubectl("-n", mrc.namespace1, "patch", "CephCluster", mrc.op.installer.ClusterName,
+		"--type=json", "-p", `[{"op": "replace", "path": "/spec/storage/storageClassDeviceSets/0/count", "value":0}]`)
+	require.NoError(mrc.op.T(), err)
+	_, err = mrc.k8sh.Kubectl("-n", mrc.namespace1, "delete", "deployments", "rook-ceph-osd-0")
+	require.Nil(mrc.T(), err)
+	err = mrc.k8sh.WaitForPodCount("app=rook-ceph-osd", mrc.namespace1, 0)
+	require.Nil(mrc.T(), err)
+	_, err = mrc.k8sh.Kubectl("-n", mrc.namespace1, "delete", "job", "-l", "app=rook-ceph-osd-prepare")
+	require.Nil(mrc.T(), err)
+	_, err = mrc.k8sh.Kubectl("-n", mrc.namespace1, "delete", "pvc", "-l", "ceph.rook.io/DeviceSet=set1")
+	require.Nil(mrc.T(), err)
+	_, err = mrc.k8sh.Exec(mrc.namespace1, "rook-ceph-tools", "ceph", []string{"osd", "down", "osd.0"})
+	require.Nil(mrc.T(), err)
+	_, err = mrc.k8sh.Exec(mrc.namespace1, "rook-ceph-tools", "ceph", []string{"osd", "out", "osd.0"})
+	require.Nil(mrc.T(), err)
+	_, err = mrc.k8sh.Exec(mrc.namespace1, "rook-ceph-tools", "ceph", []string{"osd", "purge", "0", "--yes-i-really-mean-it"})
+	require.Nil(mrc.T(), err)
+	err = mrc.op.installer.WipeDisks(mrc.namespace1, 3)
+	require.Nil(mrc.T(), err)
+}
+
+func (mrc *MultiClusterDeploySuite) checkIfRookClusterIsInstalledInOSDonPVC(deviceType string) {
+	logger.Infof("Check if OSD on PVC backed by %q type device", deviceType)
+
+	options := metav1.ListOptions{LabelSelector: "app=rook-ceph-osd"}
+	pods, err := mrc.k8sh.Clientset.CoreV1().Pods(mrc.op.namespace1).List(options)
+	require.Nil(mrc.T(), err)
+	if len(pods.Items) >= 1 {
+		mrc.deleteOSD()
+	}
+	mrc.createOSD(deviceType)
+	checkIfRookClusterIsInstalled(mrc.Suite, mrc.k8sh, installer.SystemNamespace(mrc.namespace1), mrc.op.namespace1, 1)
+	checkIfRookClusterIsHealthy(mrc.Suite, mrc.testClient, mrc.namespace1)
+}
+
 // Test to make sure all rook components are installed and Running
 func (mrc *MultiClusterDeploySuite) TestInstallingMultipleRookClusters() {
```

Comment:
How about we name the test according to testing the OSDs?
```suggestion
func (mrc *MultiClusterDeploySuite) TestOSDConfigurationOnPVCs() {
```

Reply:
Sounds good, will change.